shopify_features/models/profiles.yaml

```diff
@@ -7,20 +7,236 @@ models:
       entity_key: user
       main_id_type: main_id
       edge_sources:
-        - inputs/rsIdentifies
-        - inputs/rsTracks
-        - inputs/rsPages
-        - inputs/rsOrderCreated
-        - inputs/rsOrderCancelled
-        - inputs/rsCartUpdate
-        - inputs/rsCartCreate
-        - inputs/rsTracksUnionPages
-        - inputs/rsCartCreateUnionCartUpdate
-        - inputs/rsCART_LINE_ITEMS
-        - inputs/rsLAST_CART_STATUS
-        - inputs/rsORDER_CREATED_ORDER_CANCELLED
-        - inputs/rsITEMS_PURCHASED_EVER
-
+        - from: inputs/rsIdentifies
+        - from: inputs/rsTracks
+        - from: inputs/rsPages
+        - from: inputs/rsOrderCreated
+        - from: inputs/rsOrderCancelled
+        - from: inputs/rsCartUpdate
+        - from: inputs/rsCartCreate
+   
+  - name: CartCreateUnionCartUpdate
+    model_type: sql_template
+    model_spec:
+      validity_time: 24h # 1 day
+      materialization:
+        output_type: ephemeral
+        run_type: discrete
+      single_sql: |
+        {% with CartCreate = this.DeRef("inputs/rsCartCreate") CartUpdate = this.DeRef("inputs/rsCartUpdate") %}
+            select anonymous_id,timestamp,token from {{CartCreate}}
+            union all 
+            select anonymous_id,timestamp,token from {{CartUpdate}}
+        {% endwith %}
+      ids:
+    
+        - select: "anonymous_id"
+          type: anonymous_id
+          entity: user
+    
+  - name: TracksUnionPages
+    model_type: sql_template
+    model_spec:
+      validity_time: 24h # 1 day
+      materialization:
+        output_type: ephemeral
+        run_type: discrete
+      single_sql: |
+        {% with Tracks = this.DeRef("inputs/rsTracks") Pages = this.DeRef("inputs/rsPages") %}
+            select user_id, anonymous_id, context_session_id, timestamp from (
+            select ANONYMOUS_ID,USER_ID,timestamp::date as dt, min(timestamp) as timestamp,CONTEXT_SESSION_ID from {{Tracks}} group by user_id, anonymous_id, context_session_id, timestamp::date
```

Comment:
> select ANONYMOUS_ID,USER_ID,timestamp::date as dt, min(timestamp) as timestamp,CONTEXT_SESSION_ID

Can you move min(timestamp) and max(timestamp) after context_session_id? It's common pattern to have aggregates at end of selected fields;